internal/cli/cmd/kubeblocks/status.go

```diff
@@ -418,6 +419,55 @@ func (o *statusOptions) showWorkloads(ctx context.Context, allErrs *[]error) {
 	tblPrinter.Print()
 }
 
+func (o *statusOptions) showK8sClusterInfos(ctx context.Context, allErrs *[]error) {
+	version, err := util.GetVersionInfo(o.client)
+	if err != nil {
+		appendErrIgnoreNotFound(allErrs, err)
+	}
+	if o.ns != "" {
+		fmt.Fprintf(o.Out, "KubeBlocks is deployed in namespace: %s", o.ns)
+		if version.KubeBlocks != "" {
+			fmt.Fprintf(o.Out, ",version: %s\n", version.KubeBlocks)
+		} else {
+			printer.PrintBlankLine(o.Out)
+		}
+	}
+
+	provider, err := util.GetK8sProvider(version.Kubernetes, o.client)
+	if err != nil {
+		*allErrs = append(*allErrs, fmt.Errorf("failed to get kubernetes provider: %v", err))
+	}
+	if !provider.IsCloud() {
```

Comment:
can we guarantee `provider` is never nil even on error?

Reply:
> can we guarantee `provider` is never nil even on error?

`provider` is type of string and even error the  the `provider` will be `unknown` due `internal/cli/util/provider.go  --  func GetK8sProvider()` and the IsCloud() return false.